bloom/commands/git/import_upstream.py

```diff
@@ -205,6 +206,12 @@ def handle_tree(tree, directory, root_path, version):
                 file_data = show(BLOOM_CONFIG_BRANCH, os.path.join(root_path, rel_path))
             # Write file
             with open(rel_path, 'wb') as f:
+                # Python 2 will treat this as an ascii string but
+                # Python 3 will not re-decode a utf-8 string.
+                try:
+                    file_data = file_data.encode('utf-8')
+                except UnicodeDecodeError:
+                    file_data = file_data.decode('utf-8').encode('utf-8')
```

Comment:
A minimal representation of the problem in py2. Note that if you enter the string as `"Ragnarök"` it works fine because something about the interactive python interpreter recognizes it as a utf-8 string already.

```python
"Ragnar\xc3\xb6k".encode('utf-8')
# Traceback (most recent call last):
# File "<stdin>", line 1, in <module>
# UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 6: ordinal not in range(128)
#
# 
"Ragnar\xc3\xb6k".decode('utf-8').encode('utf-8')
# ==> 'Ragnar\xc3\xb6k
```